src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/LockFiles/simpleApp.project.lock.json

```diff
@@ -0,0 +1,6644 @@
+{
+    "locked": false,
```

Comment:
This file is more than 6000 lines long. It looks like in the tests you're loading it and then adding the test libraries you need, so can you instead use an assets file that's as simple as possible from a project with no package references as the starting point?

Reply:
Yep, pretty much. I'm not really sure if there's a better way to do this. I found other tests that were using a different project.lock.json (and even wrote a couple tests against it), but it had some extra dependencies that I didn't want that would've just been confusing. (I think they also made it impossible to properly test a couple scenarios like "no references.") I basically just copied what was there then deleted the parts I didn't want.